### PR TITLE
AWS - Security Hub - handle a non pagination response of the 'get findings' command 

### PR DESCRIPTION
**Pull request by ShacharKidor**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/37476

## Description
Solving bug #37476 - Cannot filter events in AWS Security Hub integration. 
The bug 

## Minimum version of Cortex XSOAR
- [x] 5.5.0
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by ShacharKidor** [comment]
see failing test in: [https://app.circleci.com/pipelines/github/demisto/content/89094/workflows/a272a1bc-a9e0-41de-a664-6822c264c8b1/jobs/369066](url)

File: Packs/AWS-SecurityHub/ReleaseNotes/1_0_6.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### AWS - Security Hub
+- Fix an issue where non pagination responses of **aws-securityhub-get-findings** command returned no data.
```
Comment (bakatzir) [inline Packs/AWS-SecurityHub/ReleaseNotes/1_0_6.md:4]:
```suggestion
- Fixed an issue where the ***aws-securityhub-get-findings*** command did not return data in some cases .
```

File: Packs/AWS-SecurityHub/Integrations/AWS_SecurityHub/AWS_SecurityHub.py [hunk 2]
```diff
@@ -703,8 +703,8 @@ def get_master_account_command(client, args):
 
 def get_findings_command(client, args):
     kwargs = generate_kwargs_for_get_findings(args)
-    findings = []
     response = client.get_findings(**kwargs)
+    findings = response.get('Findings')
```
Comment (Itay4) [inline Packs/AWS-SecurityHub/Integrations/AWS_SecurityHub/AWS_SecurityHub.py:707]:
```suggestion
    findings = response.get('Findings', [])
```

File: Packs/AWS-SecurityHub/Integrations/AWS_SecurityHub/AWS_SecurityHub_test.py [hunk 3]
```diff
@@ -114,3 +117,36 @@ def test_parse_resource_ids(test_input, expected_output):
...
+
+def test_aws_securityhub_get_findings_command():
+    """
+    Given:
+        - A dictionary that represents response body of aws_securityhub_get_findings API call without pagination -
+        i.e doesn't have 'NextToken' key.
+    When:
+        - Running get_findings_command
+    Then:
+        Verify returned value is as expected - i.e the findings list.
```
Comment (Itay4) [inline Packs/AWS-SecurityHub/Integrations/AWS_SecurityHub/AWS_SecurityHub_test.py:141]:
```suggestion
        - Verify returned value is as expected - i.e the findings list.
```

File: Packs/AWS-SecurityHub/Integrations/AWS_SecurityHub/AWS_SecurityHub_test.py [hunk 4]
```diff
@@ -114,3 +117,36 @@ def test_parse_resource_ids(test_input, expected_output):
     """
     from AWS_SecurityHub import parse_resource_ids
     assert parse_resource_ids(test_input) == expected_output
+
+
+class MockClient:
+
+    def get_findings(self, **kwargs):
+        return {'Findings': [
```
Comment (Itay4) [inline Packs/AWS-SecurityHub/Integrations/AWS_SecurityHub/AWS_SecurityHub_test.py:125]: better to set the findings object as a const so you wont have to initialize it here and in the test itself

**Review by Itay4 — APPROVED** [review]
looks good 👍 
added a few minor comments

**Pull request merged**